core/templates/services/code-normalizer.service.ts

```diff
@@ -94,7 +95,8 @@ export class CodeNormalizerService {
       }
 
       if (isShortfallLine) {
-        existingNumSpaces = Object.keys(numSpacesToDesiredIndentLevel);
+        // eslint-disable-next-line max-len
+        existingNumSpaces = Object.keys(numSpacesToDesiredIndentLevel).map(Number);
```

Comment:
We prefer to limit the usage of `eslint-disable` exceptions. You can do something like this:
```suggestion
        existingNumSpaces = Object.keys(
          numSpacesToDesiredIndentLevel
        ).map(Number);
```